codeship_yaml/main.py

```diff
@@ -49,17 +50,50 @@ def exec_section(self, section_name):
             commands = []
 
         for cmd in commands:
+            log('========== Running: {}'.format(cmd), color='blue')
+            start = time.time()
             status = os.system(cmd)
+            duration = round(time.time() - start, 2)
+
+            duration_msg = '{seconds}s: {command}'.format(
+                seconds=duration, command=cmd)
+
             if status > 0:
-                fail(exit_code=status)
+                failure_message = '========== Errored after {}'.format(
+                    duration_msg)
+                fail(message=failure_message, exit_code=status)
```

Comment:
I'm really sorry to be picky, but ... can you simpy put the string into the `fail` method? No variable necessary, no named parameter either. Make it look like the `log` call below. Thanks!